modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/tree/BPlusTree.java

```diff
@@ -2169,6 +2169,20 @@ public List<L> remove(L lower, L upper, int limit) throws IgniteCheckedException
         return Collections.unmodifiableList(rmvOp.removedRows);
     }
 
+    /**
+     * @param lower Lower bound (inclusive).
+     * @param upper Upper bound (inclusive).
+     * @param x Implementation specific argument.
+     * @param limit Limit of processed entries by single call, {@code 0} for no limit.
```

Comment:
Both invocations of the method pass -1 as parameter limit. Should this value be added to the Javadoc?

Reply:
Ok